test/fuzz/basic.ml

```diff
@@ -336,3 +336,95 @@ let block_type env =
   let+ param_type = list (param env)
   and+ result_type = list val_type in
   Arg.Bt_raw (None, (param_type, result_type))
+
+let memory_size = const Memory_size
+
+let memory_grow = const Memory_grow
+
+let memory_copy = const Memory_copy
+
+let memory_fill = const Memory_fill
+
+let memory_init = const (Memory_init (Raw 0))
+
+let memarg =
+  let* offset = range 10 in
+  let+ align = range 10 in
+  { offset; align}
+
+let i32_load_i32 =
```

Comment:
Could you rename the function `i32_load` and do the same for all the following functions ? They're all going to use an i32, which is the memory offset. I'm not sure it makes thing clearer to have it in the name.

Reply:
Same with `_store_` functions